Program.cs

```diff
@@ -1,4 +1,24 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+﻿// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.                                              
+// ------------------------------------------------------------------------
+// Program.cs
+// Extension of Spelling Bee assignment
+// Display the first 7 letters to be used as the seed for the Spelling Bee program from the given word list.
+// --------------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+// Reading all letter in the wordlist.
+string wordsList = File.ReadAllText ("D:/OneDrive - Trumpf Metamation Pvt Ltd/words.txt");
+Dictionary<char, int> output = new ();
+foreach (char ch in wordsList)
+   if (ch >= 'A' && ch <= 'Z') output[ch] = output.TryGetValue (ch, out int cnt) ? cnt + 1 : 1;
+// Arrange the letters in descending order based on the value.
+var orderedLetters = output.OrderByDescending (kv => kv.Value).ToList ();
+Console.WriteLine ("Character Count in Descending Order:");
+foreach (var kvp in orderedLetters)
+   Console.WriteLine ($"Character: {kvp.Key}, Count: {kvp.Value}");

```

Comment:
Don't display `Character` and `Count` for each row. Display them as column headers instead. See an example below.
![image](https://github.com/Nandhini-Sakthi/Training/assets/75833208/66f53290-1bb9-4b23-bf1e-d681f184acdf)